src/modules/sdlog2/sdlog2_format.h

```diff
@@ -80,7 +80,7 @@ struct log_format_s {
 	uint8_t length;		// full packet length including header
 	char name[4];
 	char format[16];
-	char labels[64];
+	char labels[128];
```

Comment:
This will break replay, so will require a fix on the other end - but here is a potential fix: ```MaxI,MayI,MazI,MaxIV,MayIV,MazIV,HeI,HeIV,AiI,AiIV,BeI,BeIV```